CHANGELOG.develop

```diff
@@ -138,6 +138,8 @@ Benner and Paweł Siudak):
 **** Syntax checking
 - Key bindings (thanks to Loris Lucido):
   - Changed ~SPC e e~ to ~SPC e b~ for =flycheck-buffer=
+**** Extra
+- Delete extra-langs layer (thanks to Robby O'Connor)  
```

Comment:
Believe this should go under *Removed layers*.

Reply:
> *** Removed layers
> - =extra-langs= (replaced by ad-hoc layers)
> - =evil-cleverparens= (moved to =spacemacs-evil=)

you are right, I want to delete it, because I had saw it in the doc